.github/workflows/bigquery-ded.yml

```diff
@@ -18,13 +18,29 @@ jobs:
     timeout-minutes: 20
     env:
       GCLOUD_VERSION: 290.0.1
-      BQ_PREFIX: dedicated_${{ github.event.pull_request.number }}_
       BQ_PROJECT: ${{ secrets.BQ_PROJECT_CD }}
       BQ_REGION: ${{ secrets.BQ_REGION_CD }}
       BQ_BUCKET: ${{ secrets.BQ_BUCKET_CD }}
     steps:
       - name: Checkout repo
         uses: actions/checkout@v2
+      - name: Set BQ_PREFIX for releases
```

Comment:
I'm not sure if we want this for the dedicateds. Maybe this was defined as it is. Normally a dedicated consist of deploying temporally our UDFs in a controlled env so we can test it before merging and then just remove that temp env. Even if we have an approach for hot fixes I believe we will need dedicateds still for QA

Reply:
I am not sure I understand the problem. This was not defined, but I just thought it would be more descriptive to deploy releases and hotfixes with different names like 'dedicated_release_version_' instead of 'dedicated_prNumber_' like the regular dedicated schemas.
The regular dedicated will continue working exactly like they were, or that's the idea at least

Reply:
I see what you mean. I'm worried then if two of us are working in different PRs and want just to do some tests and update the version once everything is ready we would have a conflict I guess. PR_Number was a way to have totally independent dedicateds. If you think there is no possibility of conflicts then it LGTM.